### DISCUSSION TITLE
[Bug]: Update Button is grayed out.  Will not update.

### DISCUSSION THREAD

**Discussion by ebart455987** [code4recovery/12-step-meeting-list#691] [Q&A] [answered]
### Contact Details

wm@fortworthaa.org

### Website With Issue

www.fortworthaa.org

### What happened?

Update Button is grayed out.  Will not update.
Using  WP version 5.9
TSML Version 3.12.2
Tried using Safari and Chrome

Add and Trash Meetings works.  

Thanks for everything you do.

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome, Safari

**Comment by kiyote33** [comment]
Hi @ebart455987,

Thanks for reporting this issue. It appears to be the same issue that was reported last week (#663) and a fix is in the works with an expected release soon. Please let us know if our next version update does not resolve this issue for you and we will address it immediately.

Please note: For our work flow, we’re asking that user issues be opened inside of Discussions first. The maintainers will then migrate the details to an Issue when appropriate. Thanks!

**Reply by ebart455987** [reply]
Thank You for the quick response.

**Comment by brianw-area24** [comment, marked as answer]
Had another site this work on, so thought I'd throw out another suggestion. In the Imports & Settings page, what do you have for Address Geocoding Method? If you have the Beta - API Gateway, lets try changing that to Legacy and see if that helps.

Brian W.

**Reply by ebart455987** [reply]
THAT WORKED!!!  THANKS!!!   I can get back to making updates to the meeting list.
